core/client/app/services/time-zone.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+const {
+    Service,
+    computed,
+    inject: {service}
+} = Ember;
+
+export default Service.extend({
+    store: service(),
+
+    _parseTimezones(settings) {
+        let activeTimezone = settings.get('activeTimezone');
+        try {
+            let [ , offset ] = activeTimezone;
+            return offset;
+        } catch (e) {
```

Comment:
Did you run into errors whilst developing that meant you need the `catch` statement here?

Generally when catching errors it's a good idea to handle only the specific errors that you expect to happen and re-throw the error if it's something unexpected, or at least document with a comment about why an expected error may occur. E.g:

```javascript
try {
  x.process();
} catch(e) {
    if (e instanceof ExpectedError) {
        // `x` will not be processable if the user hasn't chosen a processor,
        // return a blank hash so that later code doesn't fail
        return {};
    }

    throw e;
}
```

The above would mean that unexpected errors are not silently swallowed and other developers aren't left guessing at what the known error states could be.